src/plugins/cordova-plugin-device/sim-host.js

```diff
@@ -429,12 +429,17 @@ function registerTelemetryEvents() {
         }));
     });
 
-    // Register event for the isVirtual device checkbox
-    document.getElementById('is-virtual-device')
-        .addEventListener('click',
-            telemetry.sendUITelemetry.bind(this, Object.assign({}, baseProps, {
-                control: 'is-virtual-device'
-            })));
+    // Register event for the isVirtual device checkbox. Clicking the checkbox's label fires the click event twice, so
+    // keep track of the previous state. Note that we can't use the change event because the component seems to
+    // swallow it.
+    var previousVirtualState = true;
```

Comment:
Unrelated telemetry change; the `click` event on `cordova-checkbox` components gets fired twice when clicking the checkbox's label. This logic is needed to work around that.